primitives/api/test/tests/runtime_calls.rs

```diff
@@ -167,6 +166,7 @@ fn record_proof_works() {
 	}
 	.into_signed_tx();
 
+	let block_id = BlockId::Hash(client.chain_info().best_hash);
```

Comment:
```suggestion
	let best_hash = BlockId::Hash(client.chain_info().best_hash);
```

Reply:
Typically `block_id` was always called `block_id`. Please also note that its type is not a `Hash`, it is a `BlockId`.